test/CadlRanchProjects.Tests/azure-core-basic.cs

```diff
@@ -129,5 +131,19 @@ await foreach (Page<SecondItem> page in second.AsPages())
                 Assert.AreEqual("Madge", firstItem.Name);
             }
         });
+
+        [Test]
+        public void Azure_Core_basic_RenameGetMethod()
+        {
+            var method = typeof(BasicClient).GetMethod("GetUserAsync", new[] { typeof(int), typeof(CancellationToken) });
```

Comment:
Can we add a line testing there is no method called `Get` ? Same as below, checking there is not method called `List`.